text/3-entrees/off-chain-workers.md

```diff
@@ -0,0 +1,327 @@
+# Off-chain Workers
+
```

Comment:
Link to pallet directory on github should go here (right after title). See other recipes' texts for example.